storage/remote/otlptranslator/prometheusremotewrite/metrics_to_prw_test.go

```diff
@@ -28,41 +28,73 @@ import (
 	"go.opentelemetry.io/collector/pdata/pmetric"
 	"go.opentelemetry.io/collector/pdata/pmetric/pmetricotlp"
 
+	"github.com/prometheus/prometheus/model/labels"
 	"github.com/prometheus/prometheus/prompb"
 	prometheustranslator "github.com/prometheus/prometheus/storage/remote/otlptranslator/prometheus"
 )
 
 func TestFromMetrics(t *testing.T) {
-	t.Run("successful", func(t *testing.T) {
-		converter := NewPrometheusConverter()
-		payload := createExportRequest(5, 128, 128, 2, 0)
-		var expMetadata []prompb.MetricMetadata
-		resourceMetricsSlice := payload.Metrics().ResourceMetrics()
-		for i := 0; i < resourceMetricsSlice.Len(); i++ {
-			scopeMetricsSlice := resourceMetricsSlice.At(i).ScopeMetrics()
-			for j := 0; j < scopeMetricsSlice.Len(); j++ {
-				metricSlice := scopeMetricsSlice.At(j).Metrics()
-				for k := 0; k < metricSlice.Len(); k++ {
-					metric := metricSlice.At(k)
-					promName := prometheustranslator.BuildCompliantName(metric, "", false, false)
-					expMetadata = append(expMetadata, prompb.MetricMetadata{
-						Type:             otelMetricTypeToPromMetricType(metric),
-						MetricFamilyName: promName,
-						Help:             metric.Description(),
-						Unit:             metric.Unit(),
-					})
+
```

Comment:
[Nit] I'd prefer if you avoid adding this blank line at the function's beginning :)
```suggestion
```